docs/source/getting_started/v1_user_guide.md

```diff
@@ -0,0 +1,132 @@
+# vLLM V1 User Guide
+
```

Comment:
Add a note in the front saying the following
"V1 is enabled by default for supported use cases! We will gradually enable V1 for all use cases we plan to support. Please leave us feedback in GitHub or in vLLM Slack"